docs/src/introduction.md

```diff
@@ -39,10 +39,11 @@ p4est_connectivity_t *p4est_connectivity_new_periodic (void);
 ```
 
 [P4est.jl](https://github.com/trixi-framework/P4est.jl) wraps these C
-functions accordingly and works with pointers. We also follow the naming
-scheme of [`p4est`](https://github.com/cburstedde/p4est). For example,
-we use `connectivity::Ptr{p4est_connectivity}`. However, it is sometimes
-useful/required to also load the wrappers of the C `struct`s from their
+functions accordingly and works with pointers. For a convenient alternative
+to using pointers, see the [next section](@ref pointer_wrappers). We also
```

Comment:
```suggestion
to using raw pointers directly, see the next section on [`PointerWrapper`s](@ref pointer_wrappers). We also
```
Suggestion: This gives a little more context on what to expect from the next section.